roles/network_plugin/calico/templates/calico-cr.yml.j2

```diff
@@ -11,3 +11,13 @@ rules:
       - nodes
     verbs:
       - get
+{% if podsecuritypolicy_enabled %}
```

Comment:
what if this is added and podsecurity is not enabled ? will it be ignored? if yes I'd remove the condition

Reply:
Good question I will test it right now and let you know

Reply:
It's indeed ignored. Removed the conditions in 25296e2